### PR TITLE
ICMSLST-1405 Licence & Certificate reference changes

### PR DESCRIPTION
**Pull request by MattHolmes123**

### CODE REVIEW

File: web/domains/case/_import/models.py [hunk 1]
```diff
@@ -368,6 +359,33 @@ def licence_issue_date(self):
     def get_specific_model(self) -> "ImportApplication":
         return super().get_specific_model()
 
+    def get_most_recent_licence(self) -> "ImportApplicationLicence":
+        return self.licences.filter(
+            status__in=[
+                ImportApplicationLicence.Status.DRAFT,
+                ImportApplicationLicence.Status.ACTIVE,
+            ]
+        ).order_by("-created_at")[0]
```
Comment (mikeylockwood) [inline web/domains/case/_import/models.py:368]: I tend to use `.first()` instead of [0] for django queries

Comment (MattHolmes123) [reply web/domains/case/_import/models.py:368]: I do when it's optional.
its a bug if there isn't one and I want the system to fail

File: web/templates/web/domains/case/import/manage/preview-licence.html [hunk 2]
```diff
@@ -136,8 +136,9 @@
             <br />
             <br />
             This licence will cease to be valid for importations after
-            {% if process.licence_end_date %}
-              {{ process.licence_end_date.strftime("%d %B %Y") }}.
+            {% if {{ licence_end_date }} %}
+              {# {{ licence_end_date.strftime("%d %B %Y") }} .#}
```
Comment (mikeylockwood) [inline web/templates/web/domains/case/import/manage/preview-licence.html:140]: Commented out?

Comment (MattHolmes123) [reply web/templates/web/domains/case/import/manage/preview-licence.html:140]: Yes for now.
preview-licence.html is the old template that is a load of rubbish.
It is only used so preview licence doesn't error for applications that haven't implemented it yet.

**Review by mikeylockwood — APPROVED** [review]

**Pull request merged**